src/pages/createPost.tsx

```diff
@@ -154,122 +62,111 @@ const createPost = () => {
+                <LocalizationProvider dateAdapter={AdapterDayjs}>
+                  <DateTimePicker
+                    value={value}
+                    onChange={handleChange}
+                    renderInput={(params) => <TextField {...params} fullWidth />}
+                  />
+                </LocalizationProvider>
                 <div style={helperTextBox}>
                   <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
                 </div>
-            
-            </Grid> 
-
-          <Grid item xs={6}>
+              </Grid>
+              <Grid item xs={6}>
                 <Typography variant="body1">End</Typography>
                 <LocalizationProvider dateAdapter={AdapterDayjs}>
-              <DateTimePicker
-          
-          value={value}
-          onChange={handleChange}
-          renderInput={(params) => <TextField {...params} fullWidth />}
-          
-        /></LocalizationProvider>
-                  <div style={helperTextBox}>
-                    <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
-                  </div>
-            
-          </Grid> 
-
-        </Grid>
-
+                  <DateTimePicker
+                    value={value}
+                    onChange={handleChange}
+                    renderInput={(params) => <TextField {...params} fullWidth />}
+                  />
+                </LocalizationProvider>
+                <div style={helperTextBox}>
+                  <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
+                </div>
+              </Grid>
+            </Grid>
           </Grid>
-
-
           <Grid item>
             <Typography variant="h5">Tag</Typography>
             <Autocomplete
-            multiple
-            limitTags={5}
-            id="multiple-limit-tags"
-            options={Tags}
-            getOptionLabel={(option) => option.Tag}
-            renderInput={(params) => (
-              <TextField {...params} placeholder="Click for add tag" fullWidth />
-            )}
-            
+              multiple
+              limitTags={5}
+              id="multiple-limit-tags"
+              options={Tags}
+              getOptionLabel={(option) => option.Tag}
+              renderInput={(params) => (
+                <TextField {...params} placeholder="Click for add tag" fullWidth />
+              )}
             />
-            
-            
           </Grid>
-          
-
-
-
           <Grid item>
-          <Typography variant="h5">Description</Typography>
-          <TextField variant="outlined" fullWidth size="Big" placeholder="เช่น มาเที่ยวกันเลย ร้านบอร์ดเกมแถวรัชดา" />
+            <Typography variant="h5">Description</Typography>
+            <TextField
+              variant="outlined"
+              fullWidth
+              size="medium"
+              placeholder="เช่น มาเที่ยวกันเลย ร้านบอร์ดเกมแถวรัชดา"
+            />
```

Comment:
This text field should larger than the others and display scroll when text is overflow. Please fix it